recipes/code-aster/meta.yaml

```diff
@@ -0,0 +1,87 @@
+{% set name = "aster" %}
+{% set version = "14.6.0" %}
+#{% set sha256 = "2afa430a588dc690c7797e3b87de67742cd19a92c82bcb715b007709b0983131" %}
+{% set sha256 = "dcb3900de1e5c0624060253660a63512ab415cbe143cdd87867a0276ef6bf233" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}_{{ version }}.tar.gz
+  #url: https://www.code-aster.org/FICHIERS/aster-full-src-14.6.0-1.noarch.tar.gz # Downloaded bytes did not match Content-Length
+  url: https://collegamenti.s3.eu-west-1.amazonaws.com/aster-full-src-{{ version }}-1.noarch.tar.gz
+  sha256: {{ sha256 }}
+    #git_url: https://sourceforge.net/projects/codeaster/
+    #git_rev: 
+
+build:
+  number: 0
+  skip: true  # [py2k or osx or win]
+  detect_binary_files_with_prefix: True
+ # has_prefix_files:
+ #   - {{ version }}/share/aster/profile.sh
+ #   - etc/codeaster/profile.sh
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - sysroot_linux-64 =2.17
+  host:
+    - make
+    - bison 
+    - flex 
+    - swig 
+    - gettext  
+    - tk   
+    - boost {{ boost }}
+    - python {{ python }}
+    - numpy {{ numpy }}
+    - mfront 3.2.1
+    #- metis=5.1.0    
+    #- mumps-seq=5.1.2    
```

Comment:
Please clean unexplained commented out dependencies and other lines from your recipes.